Rl add quizz

Kurkos opened the pull request:
REFERENCE: AddQuizz component + part of AddQuizzContainer;
DESCRIPTION: Finished working on AddQuizz componnents(without REDUX connection);

janisknets commented on src/index.js:
```diff
@@ -6,6 +6,7 @@ import "bootstrap/dist/css/bootstrap.min.css";
 import App from "./App";
 import Login from "./components/Login/Login";
 import * as serviceWorker from "./serviceWorker";
+import AddQuizz from "./components/AddQuizz/AddQuizz";
```
You have to import the container here and use the component in the Container

Kurkos committed:
Merge branch 'develop' of https://github.com/rokija/quiz-game into RL_AddQuizz
+ changed Container name;

Kurkos committed:
Merge branch 'develop' of https://github.com/rokija/quiz-game into RL_AddQuizz

Kurkos committed:
-

rokija commented on src/containers/QuizzContainer.js:
```diff
@@ -0,0 +1,38 @@
+            description: '',
+        };
+    }
+    titleInputChange = (e) => this.setState({ title: e.target.value });
```
onTitleInputChange

rokija commented on src/containers/QuizzContainer.js:
```diff
@@ -0,0 +1,38 @@
+    }
+    titleInputChange = (e) => this.setState({ title: e.target.value });
+
+    descriptionInputChange = (e) => this.setState({ description: e.target.value });
```
onDescriptionInputChange

rokija commented on src/App.js:
```diff
@@ -1,7 +1,8 @@
 import { Link } from "react-router-dom";
 import "./App.css";
-import MainLayout from "./components/MainLayout/MainLayout";
+import { Link } from "react-router-dom"
```
its imported twice

Kurkos committed:
REFERENCE:AddQuizz fix;
DESCRIPTION:Did requested changes + removed unnecassary elements + added Main Layout;

Kurkos committed:
Merge branch 'develop' of https://github.com/rokija/quiz-game into RL_AddQuizz
+ deleted MainLAyout from AddQuizz component;

Kurkos committed:
Fix
